tests/Support/SupportCollectionTest.php

```diff
@@ -778,6 +778,13 @@ public function testGettingMinItemsFromCollection()
         $c = new Collection();
         $this->assertNull($c->min());
     }
+    
+	public function testPrepend()
```

Comment:
Code style. Please follow PSR-2 and some other rules as defined in our .phpcs file. If you're unsure what they mean, go and read the fixer definitions on php-cs-fixer.

In this specific case, you need a single space between the closing bracket and the dollar for the array fast.